packages/devtools/src/constants.js

```diff
@@ -125,7 +125,6 @@ export const BROWSER_ERROR_MESSAGES = {
 export const VENDOR_PREFIX = {
     chrome: 'goog:chromeOptions',
     firefox: 'moz:firefoxOptions',
-    edge: 'ms:edgeOptions'
```

Comment:
Chromium Edge is supported too!

Reply:
let's talk about it in one place https://github.com/webdriverio/webdriverio/pull/5943#discussion_r500648714

Reply:
We still need this to properly generated [client capabilities](https://github.com/webdriverio/webdriverio/blob/master/packages/devtools/src/index.js#L40-L59)

Reply:
Assuming that user won't ever use "ms:edgeOptions" with devtools why would we need it?
Was not able to figure out the use case

Reply:
@christian-bromann reverted this one, however still don't understand why we need it.